<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="jobs_tab">
       <attribute name="title">
        <string>Jobs</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QScrollArea" name="running_scroll">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="running_w">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>762</width>
              <height>312</height>
             </rect>
            </property>
           </widget>
          </widget>
          <widget class="QScrollArea" name="pending_scroll">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="pendding_w">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>762</width>
              <height>178</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="hosts_tab">
       <attribute name="title">
        <string>Hosts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>Hostname</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuControl">
    <property name="title">
     <string>Control</string>
    </property>
    <addaction name="actionRescedule"/>
    <addaction name="actionDelete"/>
    <addaction name="actionSuspend"/>
    <addaction name="actionResume"/>
    <addaction name="actionHold"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuSettings"/>
   <addaction name="menuControl"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionRescedule">
   <property name="text">
    <string>Rescedule</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionSuspend">
   <property name="text">
    <string>Suspend</string>
   </property>
  </action>
  <action name="actionResume">
   <property name="text">
    <string>Resume</string>
   </property>
  </action>
  <action name="actionHold">
   <property name="text">
    <string>Hold</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
